index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Set up and Mobile Version Skeletion</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+  <header>
+    <div class="nav-bar">
+      <a href="#">
+        My Logo
+      </a>
+      <i class="fas fa-bars"></i>
```

Comment:
- [ ] What do you think about putting the Hamburger in a `nav` tag? Nav tags are semantic tags and improve your web pages' overall semantic structure, SEO score, and accessibility. They usually enclose elements that navigate you around the page like mobile menus typically do. Please do implement this.